lib/CL/devices/ventus/pocl_ventus.cc

```diff
@@ -1101,6 +1101,31 @@ void pocl_ventus_write(void *data,
   assert(0 == err);
 }
 
+void
+pocl_ventus_driver_copy (void *data, pocl_mem_identifier *dst_mem_id, cl_mem dst_buf,
+                  pocl_mem_identifier *src_mem_id, cl_mem src_buf,
+                  size_t dst_offset, size_t src_offset, size_t size)
+{
+    vt_device_data_t *d = (vt_device_data_t *)data;
+    cl_mem_flags flags = dst_buf->flags;
```

Comment:
Where is "flags" used? Or is it more appropriate to add the conditions of "CL_MEM_USE_HOST_PTR & dst_buf->flags"?

Reply:
The arraycopy use case has been verified to have no impact.